package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "avocado",
   "jest": {
+    "automock": false,
+    "setupFiles": [
+      "./setupJest.js"
+    ],
     "verbose": true,
     "setupFilesAfterEnv": [
-      "<rootDir>src/App.test.js"
+      "<rootDir>src/tests/ExperimentList.test.js"
```

Comment:
I think we don't need this setting here at all actually, try removing it and seeing if stuff still passes.